testnets/nobletestnet/assetlist.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "noble",
```

Comment:
should be 'nobletestnet'

Reply:
https://github.com/strangelove-ventures/chain-registry/commit/cb3b78bf3125b3bc6842fb09be2bc179b0181caf